src/client/opentelemetry_client_ocurl.ml

```diff
@@ -200,6 +201,9 @@ let mk_emitter ~(config:Config.t) () : (module EMITTER) =
 
   let ((module C) as curl) = (module Curl() : CURL) in
 
+  let on_tick_cbs_ = ref (ref []) in
```

Comment:
Haven't quite understood why this is a double `ref`, but I'm sure there's a good reason :)

Reply:
for initialization reasons, yeah…